src/Persistence/MassTransit.DynamoDbIntegration/DynamoDbIntegration/Saga/DynamoDbDatabaseContext.cs

```diff
@@ -90,75 +93,19 @@ async Task Save(TSaga instance)
             }, _options.DefaultConfig());
```

Comment:
Save needs the same thing, check to ensure an existing saga instance doesn't already exist.